src/components/swap/TradePrice.tsx

```diff
@@ -33,7 +33,14 @@ export default function TradePrice({ price }: TradePriceProps) {
   const { baseCurrency, quoteCurrency } = price
   const { data: usdPrice } = useUSDPrice(tryParseCurrencyAmount('1', showInverted ? baseCurrency : quoteCurrency))
 
-  const formattedPrice = formatPrice(showInverted ? price : price.invert(), NumberType.TokenTx)
+  let formattedPrice: string
+  try {
+    formattedPrice = showInverted
+      ? formatPrice(price, NumberType.TokenTx)
```

Comment:
With this happening across mobile and web, and it being an expected occurrence, can you add a ticket and comment to update `formatPrice` in `conedison` to return '0' and log the divide by 0 instead of throwing?

Reply:
the problem isn't the `formatPrice` its actually `price.invert()` that is throwing